pytorch_lightning/utilities/cli.py

```diff
@@ -277,69 +273,6 @@ def set_choices(self, nested_key: str, classes: Tuple[Type, ...], is_list: bool
         """
         self._choices[nested_key] = (classes, is_list)
 
-    @staticmethod
-    def _convert_argv_issue_84(classes: Tuple[Type, ...], nested_key: str, argv: List[str]) -> List[str]:
```

Comment:
Might also want to update `_convert_argv_issue_85`, as it includes logic to support "shorthand" notation

Reply:
I have looked at `_convert_argv_issue_85` but don't see any benefit in updating things there. Better to leave it like that until it can be removed.